hashes/polynomial_rolling_hash.py

```diff
@@ -0,0 +1,33 @@
+"""
+Polynomial Rolling Hash Algo -->
+Polynomial rolling hash function is a hash function that uses only multiplications and additions.
```

Comment:
```suggestion
Polynomial rolling hash function is a hash function that uses only multiplications
and additions.
```

Reply:
It will rectify ruff issue.

Reply:
how can i solve that give me guidance

Reply:
Just commit my suggestion.

Reply:
can you plz explain again

Reply:
![Screenshot_20230926-222953](https://github.com/TheAlgorithms/Python/assets/96521078/d44c7508-4e1c-4c33-8f5b-aafb9afd8970)
This suggestion to break 3rd line into 2 lines as length of 3rd is more than required.

Reply:
okk sir I'll resolve it by tomorrow..
can I request you to be my mentor for this hacktober fest ?!

Reply:
It is already resolved now as no ruff and pre-conmit check is failing.

Sure :)

Reply:
so now my PR will get merged or not ?
& sir i requested a connection on linkedIn... please accept

Reply:
I and community members will review the PR to ensure it meets repo standards. If any errors are found during the review, we'll make the necessary corrections before considering merging it.

Reply:
ohhh okokk actually I'm contributing 1st time :)
i don't have any idea :(